package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -0,0 +1,337 @@
+"""
+Secondary structure assignment (helix, sheet and loop) --- :mod:`MDAnalysis.analysis.dssp`
+=========================================================================================
+
+:Author: Egor Marin
+:Year: 2024
+:Copyright: LGPL v2.1+
+
+.. versionadded:: 2.8.0
+
+The module contains code to build hydrogend bond contact map,
+and use it to assign protein secondary structure (:class:`DSSP`).
+
+This module uses the python version of the original algorithm by Kabsch & Sander (1983),
+re-implemented by @ShintaroMinami and available under MIT license
+[here](https://github.com/ShintaroMinami/PyDSSP/tree/master#differences-from-the-original-dssp)
```

Comment:
Let's not use "here" links. Use a descriptive text like 
```suggestion
from [ShintaroMinami/PyDSSP](https://github.com/ShintaroMinami/PyDSSP/tree/master#differences-from-the-original-dssp)
```